internal/clients/http/gcp/gcp_client.go

```diff
@@ -0,0 +1,91 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+
+	computepb "google.golang.org/genproto/googleapis/cloud/compute/v1"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	"github.com/RHEnVision/provisioning-backend/internal/clients"
+	"github.com/RHEnVision/provisioning-backend/internal/ctxval"
+	"github.com/aws/smithy-go/ptr"
+	"github.com/rs/zerolog"
+)
+
+type gcpClient struct {
+	instancesClient *compute.InstancesClient
+	context         context.Context
+	logger          zerolog.Logger
+}
+
+func init() {
+	clients.GetGCPClient = newGCPClient
+}
+
+func newGCPClient(ctx context.Context) (clients.GCP, error) {
+	logger := ctxval.Logger(ctx).With().Str("client", "gcp").Logger()
+
+	instancesClient, err := compute.NewInstancesRESTClient(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("cannot create new instances REST client: %w", err)
+	}
+
+	return &gcpClient{
+		instancesClient: instancesClient,
+		context:         ctx,
+		logger:          logger,
+	}, nil
+
```

Comment:
I will remove that extra line in the next update 😸 
@lzap Isn't go linter suppose to warn about this?

Reply:
Go linter does not enforce these. But it would be nice if it would :)
Maybe there is an option? :)

Reply:
It's fine, it's just I can't sleep that's all ... :-)

Reply:
If you could remove it, I'd appreciate it :-)